roles/rook/defaults/main.yml

```diff
@@ -0,0 +1,153 @@
+---
+##########################
+# operator
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+configuration_directory: /opt/configuration
+
+##########################
+# k3s/metallb
+
+metallb_rook_external_IP: 192.168.16.101
+
+##########################
+# rook
+
+container_registry_rook: quay.io
+rook_ceph_image: "{{ container_registry_rook }}/ceph/ceph"
+# renovate: datasource=docker depName=quay.io/rook/ceph
+rook_ceph_image_tag: "v17.2"
+
+## template directory containing rook CRD `*.yml.j2` files, e.g. OSISM configuration directory
+rook_template_directory: "{{ role_path }}/templates"
+rook_configuration_directory: "{{ configuration_directory }}/environments/rook"
+## kubernetes namespace in which the rook cluster should be installed
+## also see https://rook.io/docs/rook/v1.13/Storage-Configuration/Advanced/ceph-configuration/?h=#using-alternate-namespaces
+rook_namespace: rook-ceph
+
+## work directory inside the osism-ansible container
+rook_work_directory: /tmp/rook/configuration
+
+## run cleanup tasks and delete cluster
+rook_cleanup: false
+
+##########################
+# rook CRD
+# have a look at `templates/*.yml.j2` to see where exaclty the variables are applied
+# for ultimate customization, change `{{ rook_template_directory }}` to point to your configuration repository
+
+## rook cluster name
+rook_cluster_name: rook-ceph
+rook_mon_count: 3
+rook_mds_count: 3
+rook_mgr_count: 3
+rook_mgr_modules:
+  - name: balancer
+    enabled: true
+  - name: status
+    enabled: true
+  - name: prometheus
+    enabled: true
+  # obtain the information about physical disks in dashboard
+  - name: rook
+    enabled: true
+rook_dashboard_enabled: true
+rook_dashboard_ssl: true
+rook_dashboard_port: 7000
+rook_dashboard_port_external: 443
+# https://rook.io/docs/rook/latest-release/Storage-Configuration/Monitoring/ceph-monitoring/
+rook_monitoring_enabled: false # TODO: enable 
+rook_metrics_disabled: false
+rook_network_encryption: true
+rook_network_compression: true
+rook_network_require_msgr2: false
+rook_network_public: "192.168.16.0/20"
+rook_network_cluster: "{{ rook_network_public }}"
+rook_crash_disabled: false
+rook_logcollector_enabled: true
+# also allow to run on kubnernetes master nodes
+rook_placement:
+  all:
+    tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule  
+rook_annotations: {}
+rook_labels: {}
+rook_resources: {}
+# do not use all nodes
+rook_storage_useallnodes: false
+# do not use all found devices
+rook_storage_usealldevices: false
+rook_storage_config_osdsperdevice: 1
+# enable device encryption
+rook_storage_config_encrypteddevice: true
+# define a device filter where to create OSDs
+rook_storage_devicefilter: ""
+# name nodes where to create OSDs
+rook_storage_nodes: []
+#  - name: "testbed-node-0"
+#  - name: "testbed-node-1"
+#  - name: "testbed-node-2"
+rook_cephblockpool_default_size: 3
```

Comment:
consider adding a `replica` word into the variable name, e.g. rook_cephblockpool_default_replica_size